### PR TITLE
Feature/eslint improvements

### PR DESCRIPTION
**Pull request by sqevani**
Update on eslint and some overall code changes/fixes to reflect those changes:

```
"extends": [
    "eslint:recommended",
    "google",
    "plugin:vue/essential"
  ],
  "plugins": [
    "vue"
  ],
  "rules": {
    "quotes": ["error", "single"],
    "max-len": ["error", 120],
    "comma-dangle": "off",
    "key-spacing": "off",
    "object-curly-spacing": "off",
    "space-before-function-paren": "off",
    "valid-jsdoc": "warn",
    "require-jsdoc": "off",
    "no-console": "warn",
    "no-unused-vars": "warn",
    "no-invalid-this": "off",
    "no-multi-spaces": "off",
    "new-cap": "off",
    "camelcase": "warn"
  }
```

All the changes in `src/` folder are simply adding the semicolon `;` at the end (which was done automatically by eslint)

Core changes are simply on `.eslintrc` and `package.json`

### CODE REVIEW

File: __mocks__/@download/blockies.js [hunk 1]
```diff
@@ -1 +0,0 @@
-export function renderIcon () {}
```
[the remark this answers is not in the document]
Comment (sadiqevani) [reply __mocks__/@download/blockies.js]: @davidyuk is there a better place to put the mock?

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
```
[the remark this answers is not in the document]
Comment (sqevani) [reply .eslintrc]: hmm, why does it need to be a .js file? its a config file, .babelrc, .eslintrc, .[anything else]rc, can be both, just I don't see the purpose of having config objects as JS files, where no logic is being added in those files.

File: TODO.md [hunk 3]
```diff
@@ -11,3 +11,7 @@
 ### Package JSON
 
 - Clean up devDependencies, things that are not in use!
+
+### Cleanup readme.md file
+
+- Need to cleanup readme file for better documentation
```
[the remark this answers is not in the document]
Comment (sqevani) [reply TODO.md]: Its more like a "Milestone" kinda thing in that TODO list, rather than an issue.

**Comment by sqevani** [comment]
@davidyuk 

updates pushed.

File: __mocks__/@download/blockies.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidyuk) [reply __mocks__/@download/blockies.js]: I can't find a better one: https://jestjs.io/docs/en/manual-mocks#mocking-node-modules
We can define one more root for Jest, but I'm don't see reason.

File: TODO.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (davidyuk) [reply TODO.md]: Github issues can also track milestones.

[the remark this answers is not in the document]
Comment (sqevani) [reply TODO.md]: If you wanna go with that, its fine for me! Just an in my opinion overkill! But I'll use that.

**Pull request merged**